src/main/java/Duke.java

```diff
@@ -1,10 +1,84 @@
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Scanner;
+
 public class Duke {
+
+    enum Ability {
+        BYE,
+        LIST,
+        MARK,
+        UNMARK,
+        TODO,
+        DELETE
+    }
+
     public static void main(String[] args) {
-        String logo = " ____        _        \n"
-                + "|  _ \\ _   _| | _____ \n"
-                + "| | | | | | | |/ / _ \\\n"
-                + "| |_| | |_| |   <  __/\n"
-                + "|____/ \\__,_|_|\\_\\___|\n";
-        System.out.println("Hello from\n" + logo);
+
+        String hello = "    ____________________________________________________________\n" +
+                "     Hello! I'm Duke\n" +
+                "     What can I do for you?\n" +
+                "    ____________________________________________________________\n";
+        System.out.println(hello);
+
+        List<Task> list = new ArrayList<Task>();
```

Comment:
Maybe you can change this variable name to tasks or something like that to make it clearer?